registry.json

```diff
@@ -5085,15 +5085,16 @@
     "github": "cryptobuffofficial" 
  },
  {
-   "ticker": "WFFL",
```

Comment:
You have removed WFFL pool. Please add VEGAS as an additional entry, not replace an existing one

Reply:
OMG! I'll resync and reapply. I'm surprised I've done that. So sorry.